Cargo.toml

```diff
@@ -42,7 +42,7 @@ temp-env = "0.3.6"
 thiserror = { version = "1", default-features = false }
 tonic = { version = "0.12", default-features = false }
 tonic-build = "0.12"
-tokio = { version = "1", default-features = false }
+tokio = { version = "1.38.0", default-features = false } #1.39 needs msrv bump to rustc 1.70
```

Comment:
trying to avoid being aggressive in `msrv` bump from 1.65 to 1.70, so pinning tokio version < 1.39. `tracing` is still on `1.63` so trying to being close to it.